src/AbstractSQLCompiler.ts

```diff
@@ -148,21 +74,23 @@ export type NumberTypeNodes = NumberNode | CountNode | UnknownTypeNodes;
 export type FieldNode = ['Field', string];
 export type ReferencedFieldNode = ['ReferencedField', string, string];
 export type BindNode = ['Bind', string, string?];
-export interface CastNode
-	extends TwoVarArgNodeType<'Cast', AbstractSqlType, string> {}
+export type CastNode = ['Cast', AbstractSqlType?, ...string[]];
```

Comment:
Is this surely an `AbstractSqlType?`? It was previously just `AbstractSqlType`.

Reply:
It was typed as `AbstractSqlType?` previously, but in reality it should be `['Cast', AbstractSqlType, string]` so I fixed both parts